proposals/dictionary-expressions.md

```diff
@@ -360,23 +436,15 @@ When the destination is an IEnumerable, we tend to think we're producing a seque
 
 What should we do here when targeting `IEnumerable<...>` *and* using `k:v` elements? Produce an ordered sequence, with possibly duplicated values?  Or produce an unordered dictionary, with unique keys?
 
-## Random open questions
-
-### Question 1
+Working group recommendation: `IEnumerable<KVP>` is not a dictionary type (as it lacks an indexer).  As such, it has sequential value semantics (and can include duplicates).  This would happen today anyways if someone did `[.. ldm]` and we do not think the presence of a `k:v` element changes how the semantics should work.
 
-If a user had a type that could be both initialized with a collection initializer *and* a dictionary initializer, what should happen?
-
-For example:
-
-```c#
-HybridType h = [kvp1, kvp2];
-```
+## Random open questions
 
-In C# 12 this might have been calling `.Add(kvp)`.  Is it ok to call the indexer now?  If there is an applicable `.Add`, should we defer to this, and only fallback to the indexer? 
+### Question: Parsing ambiguity
 
-### Question 2
+Parsing ambiguity around: `[a ? [b] : c]`
 
-Parsing ambiguity around: `a ? [b] : c`
+Working group recommendation: Use normal parsing here.  So this is would be the same as `[a ? ([b]) : (c)]` (a collection expression containing a conditional expression).  If the user wants a key_value_pair_element here they can write: `[(a?[b]) : c]`
```

Comment:
```suggestion
Working group recommendation: Use normal parsing here.  So this would be the same as `[a ? ([b]) : (c)]` (a collection expression containing a conditional expression).  If the user wants a `key_value_pair_element` here, they can write: `[(a?[b]) : c]`
```